### ISSUE TITLE
[SF][A11y][Token] Read Only tokens are not read by JAWS in Virtual cursor mode.

### ISSUE THREAD

**Issue by Ankitadoshi** [SAP/ui5-webcomponents#6877]
### Bug Description

When we have only read only tokens in a page, The token text are not read by JAWS screen reader in Virtual cursor mode. These tokens are not even navigable when we press arrow keys in Virtual Cursor mode.

### Expected Behaviour

All the content even read only should be accessible by arrow keys and read out by JAWS in Virtual Cursor mode

### Isolated Example

https://codesandbox.io/s/ui5-webcomponents-forked-swwzbi?file=/index.html

### Steps to Reproduce

1. Go to https://codesandbox.io/s/ui5-webcomponents-forked-swwzbi?file=/index.html
2. Open JAWS screen reader and switch to virtual Cursor mode
3. Try to navigate to 'Argentina' and 'Bulgaria' tokens with arrow keys

Observe that the cursor does not go to the tokens and it does not read the content of the tokens i.e. 'Argentina' and 'Bulgaria' are not read

### Priority

Medium

### UI5 Web Components Version

1.11.0

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by yanaminkova** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

When there is only `ui5-token` used in readonly mode, the component is not  focusable and it`s information is not being announced by JAWS even in VPC mode.  Could be reproduced here: https://codesandbox.io/s/ui5-webcomponents-forked-swwzbi?file=/index.html

I am not sure whether the `ui5-token` is even supposed to be used standalone. Could you please have a look at it and confirm?

When the `ui5-token` is used inside `Multi Input` everything works as expected and the visual feedback upon user interaction is provided.

Thanks and Kind regards,
Yana
